lib/middleware.js

```diff
@@ -33,10 +33,12 @@ module.exports = {
 
   writeHtmlFile(req, res, next) {
     debug("write html file");
-    if (!req.body.html)
-      next({ error: "Request must contain HTML for conversion" });
+    if (!req.body.html) {
+      return next(new Error("Request must contain HTML for conversion"));
+    }
     fs.writeFile(`${req.temp.name}/html.html`, req.body.html, (err) => {
-      if (err) res.status(500).send(err);
+      // Should we `next(err)` in this case as well?
+      if (err) return res.status(500).send(err);
```

Comment:
This seems like an anomaly, where the error is handled directly here instead of calling `next(err)`. Not really a big deal, but it would have been weird before I added the `return`, as we were sending a 500 response but also telling the middleware execution to continue by falling through to the `next()`